docs/access_rights/organization_account.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Organization account
+description: Create a specialized account for your team to encourage collaboration and manage permissions efficiently. Convert an existing account or create one from scratch.
+menuWeight: 8.1
+paths:
+    - access-rights/organization-account
+---
+
+# Organization account
+
+Apify's organization account allows you to keep your business account separate from your personal account. It enables you to manage your team members' [permissions]({{@link access_rights/list_of_permissions.md}}) and to centralize your billing.
+
+At the same time, a separate organization account means teams no longer have to share the credentials of a single personal account. You can [switch]({{@link access_rights/organization_account/how_to_use.md#in-the-apify-app}}) between your personal and organization accounts seamlessly in just two clicks.
```

Comment:
Here we need to add a large disclaimer that the Organization account is currently in "closed beta testing" available only for users with business and higher plans. If somebody is interested in joining this closed beta then he must contact us as support@apify.com